js/app-user.js

```diff
@@ -77,11 +77,20 @@ class User {
   constructor() {
     this.listeners = [];
     this.resetUser();
+    this.defaultUserData = {
```

Comment:
This should probably happen in `resetUser()`

Reply:
this comment here needs some updates https://github.com/mozilla/network-pulse/pull/926/files#diff-c63ea7b512f87e9b9c8728136d59562dR20